lib/drivers/tandemTslimDriver.js

```diff
@@ -1328,45 +1328,55 @@ module.exports = function (config) {
 
   var buildBasalRecords = function (data, records) {
     var basalRecords = filterLogEntries(
-      [PUMP_LOG_RECORDS.LID_BASAL_RATE_CHANGE],
+      [PUMP_LOG_RECORDS.LID_BASAL_RATE_CHANGE,PUMP_LOG_RECORDS.LID_NEW_DAY],
```

Comment:
Wait, why are we still including `PUMP_LOG_RECORDS.LID_NEW_DAY` in the filter since they're now being processed in `buildNewDayRecords`?